hack/local-up-cluster.sh

```diff
@@ -537,6 +537,7 @@ function start_kubelet {
         --experimental-cgroups-per-qos=${EXPERIMENTAL_CGROUPS_PER_QOS} \
         --cgroup-driver=${CGROUP_DRIVER} \
         --cgroup-root=${CGROUP_ROOT} \
+        --keep-terminated-pod-volumes=true \
```

Comment:
I figured `local-up-cluster.sh` is used for deploying dev environments where it is more likely the user will want to keep the volumes around for debug purposes.  I know that is what I would want personally.